src/Amazon.Lambda.Tools/LambdaPackager.cs

```diff
@@ -36,7 +37,7 @@ public static class LambdaPackager
             { "netcoreapp1.1", Version.Parse("1.6.1") }
         };
 
-        private static bool IsAmazonLinux2(IToolLogger logger)
+        private static bool IsAmazonLinux(IToolLogger logger)
```

Comment:
Originally I was thinking this method would need to change into something more complex like `IsCorrectAmazonLinux(IToolLogger logger, string targetFramework)`

Then we can use `targetFramework` to check if `.net6.0` expect `al2` and if `.net8.0` expect `al2023`.

But I guess as long as the customer includes `libicu` in their project file, either targetFramework will work with either Amazon Linux distribution. Is that right?

Reply:
Talking offline, if you've been running .NET 8 on AL2 without issues, then this isn't important